### PR TITLE
Updating Simorgh Release Info with launch dates for Front Pages & PGLs

### PR DESCRIPTION
**Pull request by karinathomasbbc**
**Overall change:**
Adding launch dates for Home Pages & PGLs

**Code changes:**
- Adding launch date information
- Regenerate Simorgh Release Info file
- Tidy up

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added labels to this PR for the relevant pod(s) affected by these changes
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by sareh — APPROVED** [review]
👍 Looks great. Thank you for the updates!

**Review by thekp — COMMENTED** [review]
Nice work

I noticed we are missing a few example URLs for home/front pages in the `docs/Simorgh-Release-Info.md ` : https://github.com/bbc/simorgh/blob/245b1142376264521e2e7c2a703d80f1dda6624d/docs/Simorgh-Release-Info.md  😬

**Comment by amywalkerdev** [comment]
> Nice work
> 
> I noticed we are missing a few example URLs for home/front pages in the `docs/Simorgh-Release-Info.md ` : https://github.com/bbc/simorgh/blob/245b1142376264521e2e7c2a703d80f1dda6624d/docs/Simorgh-Release-Info.md 😬

Thats because the urls to be used come from the cypress config, and we haven't been updating them recently

**Comment by karinathomasbbc** [comment]
> Nice work
> 
> I noticed we are missing a few example URLs for home/front pages in the `docs/Simorgh-Release-Info.md ` : https://github.com/bbc/simorgh/blob/245b1142376264521e2e7c2a703d80f1dda6624d/docs/Simorgh-Release-Info.md 😬

Hi Khoa, thanks for the review - the output is autogenerated by a script, which takes the config from Cypress. Until the test / live URLs have been added to the config, we won't see the URLs.

I guess I _could_ update the table manually... let me see how it goes.

File: scripts/simorghLaunchDates.js [hunk 1]
```diff
@@ -1,264 +1,349 @@
...
     mediaAssetPage: '6th Feb 2020',
+    storyPage: '',
   },
   hausa: {
-    articles: '',
-    frontPage: '',
+    articles: 'Sept 2019',
+    frontPage: '23rd April 2020',
     liveRadio: '30th Jan 2020',
     mediaAssetPage: '5th March 2020',
```
Comment (amywalkerdev) [inline scripts/simorghLaunchDates.js:81]:
```suggestion
    mediaAssetPage: '5th March 2020',
    photoGalleryPage: '24th March 2020',
```

File: scripts/simorghLaunchDates.js [hunk 2]
```diff
@@ -1,264 +1,349 @@
...
     liveRadio: '10th Jan 2020',
     mediaAssetPage: '27th Feb 2020',
+    photoGalleryPage: '12th March 2020',
+    storyPage: '',
   },
   gujarati: {
-    articles: '',
+    articles: 'Sept 2019',
     frontPage: '13th Jan 2020',
     mediaAssetPage: '6th Feb 2020',
```
Comment (amywalkerdev) [inline scripts/simorghLaunchDates.js:73]:
```suggestion
    mediaAssetPage: '6th Feb 2020',
    photoGalleryPage: '18th Feb 2020',
```

**Review by amywalkerdev — CHANGES_REQUESTED** [review]

**Comment by thekp** [comment]
> which takes the config from Cypress. Until the test / live URLs have been added to the config

I see we are getting the URL values from: https://github.com/bbc/simorgh/blob/a7fdf405fab6f3a2e9d8957c72fc880d3402ee90/cypress/support/config/settings.js

I'm happy for you to leave them as is, then when we are ready to update the config, we can also update the doc.

**Review by thekp — APPROVED** [review]
With Amy's suggestions, LGTM :)

**Comment by amywalkerdev** [comment]
I'm going to update urls in #6329

**Review by amywalkerdev — APPROVED** [review]

**Pull request merged**